src/main/java/io/jenkins/plugins/folderauth/FolderAuthorizationStrategyManagementLink.java

```diff
@@ -204,7 +209,7 @@ private void redirect() {
             folders = jenkins.getAllItems(AbstractFolder.class);
```

Comment:
Side note: Escalation to SYSTEM should not be required here, the user is already an ADMIN

Reply:
I think it should increase the speed of getting the folders. `SidACL` checks for `SYSTEM` first.
https://github.com/jenkinsci/jenkins/blob/22aa2e6e766074d11249893e3f35e0b99e20d3d0/core/src/main/java/hudson/security/SidACL.java#L46-L51